sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_data_lake_service_client.py

```diff
@@ -268,8 +275,12 @@ def create_file_system(self, file_system,  # type: Union[FileSystemProperties, s
                 :dedent: 8
                 :caption: Creating a file system in the datalake service.
         """
+        file_system_encryption_scope = kwargs.pop('file_system_encryption_scope', None)
         file_system_client = self.get_file_system_client(file_system)
-        file_system_client.create_file_system(metadata=metadata, public_access=public_access, **kwargs)
+        file_system_client.create_file_system(metadata=metadata,
+                                              public_access=public_access,
+                                              file_system_encryption_scope=file_system_encryption_scope,
```

Comment:
nit: Since this has the same name, you can just pass it through in `kwargs`. No need to pop it here and pass it through like this.

Reply:
Good point! I think this was leftover from the popping and mapping when I still had a `ServiceClientEncryptionScope` model nonsense.